lib/mac/src/noble_mac.mm

```diff
@@ -141,17 +141,12 @@
 // write(deviceUuid, serviceUuid, characteristicUuid, data, withoutResponse)
 Napi::Value NobleMac::Write(const Napi::CallbackInfo& info) {
     CHECK_MANAGER()
-    ARG4(String, String, String, Buffer /*, Boolean */)
+    ARG5(String, String, String, Buffer, Boolean)
```

Comment:
check history why it was removed and justify why it should be added back in commit message